lib/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "shared-github-actions-utils",

```

Comment:
Is this an accurate name?

Reply:
I think so, as it can be reused in other GitHub actions. However, I take the better name suggestion :)

Reply:
Ah, I thought this was the root package.json, didnt see `lib/`